docs/_includes/toc-layout.adoc

```diff
@@ -18,17 +18,19 @@ include::toc-left.adoc[]
 ====
 image::toc-left.png[Table of contents using left value]
 ====
-
-// NOTE: Setting `:toc: left` is equivalent to using the original AsciiDoc `toc2` attribute.
 // end::column[]
 
-// How does the toc-position attribute fit into this section?
-
 === Manual placement
 // tag::place[]
-The `macro` value on the toc attribute allows you to place the TOC anywhere in your document.
-To control where the TOC is rendered in your document, set the toc attribute to `macro`.
-Then, within the body of your document, use the `+toc::[]+` block macro to indicate the TOC's location.
+`toc` values of `auto`, `preamble`, and `macro` place the TOC in the body of the document.
+
+When `toc` is blank or set to `auto`, the TOC is inserted directly below the document’s title and author.
+
+When `toc` is set to `preamble`, the TOC is placed immediately below the preamble.
+It will not appear if your document does not have a preamble.
+
+To place the TOC anywhere else in your document, set the `toc` attribute to `macro`.
+Then, put the `+toc::[]+` block macro where you want the TOC to appear.
```

Comment:
Mention that you can only use the `toc::[]` macro once in your document. If the `toc::[]` macro is present, but the value of the `toc` attribute is not `macro`, the macro will be ignored (so it's safe to leave it there).